Source/WebInspectorUI/UserInterface/Views/LogContentView.css

```diff
@@ -158,11 +158,21 @@
     border-top-color: hsl(40, 100%, 90%);
 }
 
-.console-item::before {
+.console-item::before,
+.console-group::before {
     position: absolute;
     left: 4px;
     height: 12px;
     width: 12px;
+    z-index: 1;
+}
+
+.console-group::before {
+    top: 4px; // because .console-group does not have a top/bottom padding like .console-message
```

Comment:
Comments in CSS need to be wrapped in `/* */`, even for single-line comments. (`//` is not a valid comment in CSS)

Reply:
Addressed in 348f2f49dc517b2abe197f00972bf70aa92db4d8.